modules/install-ibm-cloud-configuring-the-install-config-file.adoc

```diff
@@ -0,0 +1,103 @@
+// This is included in the following assemblies:
+//
+// installing_ibm_cloud/install-ibm-cloud-installing-on-ibm-cloud.adoc
+
+[id="configuring-the-install-config-file_{context}"]
+= Configuring the install-config.yaml file
+
+This procedure is a substitute for the xref:../installing_bare_metal_ipi/ipi-install-installation-workflow.html#configuring-the-install-config-file_ipi-install-configuration-files[Configuring the install-config.yaml file] procedure in the "Setting up the environment for an OpenShift installation" section of the "Deploying installer-provisioned clusters on bare metal" documentation.
+
+The `install-config.yaml` file requires some additional details. Most of the information is teaching the installer and the resulting cluster enough about the available IBM Cloud&#174; hardware so that it is able to fully manage it. The material difference between installing on bare metal and installing on IBM Cloud is that you must explicitly set the privilege level for IPMI in the BMC section of the `install-config.yaml` file.
+
+.Procedure
+
+. Configure `install-config.yaml`. Change the appropriate variables to match the environment, including `pullSecret` and `sshKey`.
++
+[source,yaml]
+----
+  apiVersion: v1
+  baseDomain: <domain>
+  metadata:
+    name: <cluster-name>
+  networking:
+    machineCIDR: <public-cidr>
+    networkType: OVNKubernetes
+  compute:
+  - name: worker
+    replicas: 2
+  controlPlane:
+    name: master
+    replicas: 3
+    platform:
+      baremetal: {}
+  platform:
+    baremetal:
+      apiVIP: <api-ip>
+      ingressVIP: <wildcard-ip>
+      provisioningNetworkInterface: <NIC1>
+      provisioningNetworkCIDR: <CIDR>
+      hosts:
+        - name: openshift-master-0
+           role: master
+           bmc:
+             address: ipmi://10.196.130.145?privilegelevel=OPERATOR <1>
+             username: root
+             password: <password>
+           bootMACAddress: 00:e0:ed:6a:ca:b4 <2>
+           rootDeviceHints:
+             deviceName: "/dev/sda"
+       - name: <openshift-worker-0>
+           role: worker
+           bmc:
+             address: ipmi://<out-of-band-ip>?privilegelevel=OPERATOR <1>
+             username: <user>
+             password: <password>
+           bootMACAddress: <NIC1-mac-address> <2>
+           rootDeviceHints:
+             deviceName: "/dev/sda"
+  pullSecret: '<pull_secret>'
+  sshKey: '<ssh_pub_key>'
+----
++
+<1> The `bmc:address` specifically provides a `privilegelevel` configuration setting with the value set to `OPERATOR`. This is required for IBM Cloud.
+<2> Add the MAC address of the private `provisioning` network NIC for the corresponding node.
++
+[NOTE]
+====
+You can use the `ibmcloud` command line utility to retrieve the password.
+
+[source,terminal]
+----
+$ ibmcloud sl hardware detail --output JSON | jq .remoteManagementAccounts[0].password
```

Comment:
ibmcloud sl hardware detail <ID> --output JSON | jq '" \(.remoteManagementAccounts[0].password)"'

Replace `<ID>` with the ID of the node.

Reply:
Since this data is used further together with management-server-ip I'd suggest to use the command to get both
$ ibmcloud sl hardware detail `<ID>` --output JSON | jq '"\(.networkManagementIpAddress) \(.remoteManagementAccounts[0].password)"'